mail_tracking/README.rst

```diff
@@ -0,0 +1,107 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+=============
+Mail tracking
+=============
+
+This module shows email notification tracking status for any messages in
+mail thread (chatter). Each notified partner will have an intuitive icon just
+right to his name.
+
+
+Configuration
+=============
+
+If you're using a multi-database installation (with or without dbfilter option)
+where /web/databse/selector returns a list of more than one database, then
+you need to add ``mail_tracking`` addon to wide load addons list
+(by default, only ``web`` addon), setting ``--load`` option.
+For example, ``--load=web,mail_tracking``
+
+
+Usage
+=====
+
+When user sends a message in mail_thread (chatter), for instance in partner
+form, then an email tracking is created for each email notification. Then a
+status icon will appear just right to name of notified partner.
+
+These are all available status icons:
+
+.. |check| image:: http://image005.flaticon.com/25/png/16/25/25404.png
+   :width: 10px
+
+.. |dcheck| image:: http://image005.flaticon.com/1/png/16/88/88051.png
+   :width: 15px
+
+.. |error| image:: http://image005.flaticon.com/25/png/16/25/25298.png
+   :width: 10px
+
+.. |clock| image:: http://image005.flaticon.com/25/png/16/25/25692.png
+   :width: 10px
+
+.. |unknown| image:: http://image005.flaticon.com/25/png/16/25/25342.png
```

Comment:
This icons are not the real ones because I use FontAwesome (already included in bootstrap and Odoo core) but I didn't find how to insert a FontAwesome icons in a reStructuredText file.
But an image is better than thounsand words, isn't it?